pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "django-keyvaluestore"
+version = "1.0.0"
+homepage = "https://github.com/sdelements/django-keyvaluestore"
+description = "A simple key-value-store for Django"
+authors = ["Security Compass <contact@securitycompass.com>"]
+license = "BSD-3-Clause"
+readme = "README.md"
+# See https://pypi.python.org/pypi?%3Aaction=list_classifiers
+classifiers=[
+    # How mature is this project? Common values are
+    #   3 - Alpha
+    #   4 - Beta
+    #   5 - Production/Stable
+    'Development Status :: 5 - Production/Stable',
+
+    # Indicate who your project is intended for
+    'Intended Audience :: Developers',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+
+    # Pick your license as you wish (should match "license" above)
+    'License :: OSI Approved :: BSD-3-Clause',
+
+    # Supported Languages
+    'Natural Language :: English',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.12',
+    'Framework :: Django',
+]
+packages = [
+  { include = "keyvaluestore" },
+  { include = "tests", format = "sdist" },
+]
+exclude = [
+  "keyvaluestore/**/tests",
+  "tests"
+]
+
+[tool.poetry.dependencies]
+python = "~3.12"
+django = "~4.2"
```

Comment:
This package is technically an opensource project. We don't have any policies around how we manage these projects, i.e. what range of versions we should support instead of pinning to just version we use. It might be good to discuss this so we have something to reference going forward (even if the decision is to just pin for our usage)

Reply:
The code itself relies on django imports that's why I added this, but i'm happy to loosen the restrictions if need be?

Reply:
Chatted about this, will leave it pinned to the versions we use for now but can revisit in later discussions with eng leadership